### PR TITLE
Get build passing in travis & begin moving documentation to roxygen format

### PR DESCRIPTION
**Pull request by georgestagg**
In the DESCRIPTION file `HenrikBengtsson/matrixStats@develop` has been added under `Remotes:`, so as to pick up a bugfix that lets this build without error in travis.

Presumably the develop branch is unstable and so this should be removed and `matrixStats (>= 0.50.XX)` added back into DESCRIPTION once a new version of matrixStats containing the changes that fix the problem hits CRAN.

### CODE REVIEW

**Comment by csgillespie** [comment]
Looks good. Two minor comments. For the roxygen tags, would you use `importFrom` instead of `import` to avoid namespace clashes. Also use `inheritParams` to avoid duplication.

**Comment by georgestagg** [comment]
No problem, I've gone through and changed `import` to `importFrom` and used `inheritParams` where possible. 

I noticed during the change that nlme was imported but not used anywhere, perhaps something was changed in a previous commit and it was left behind?

I've removed nlme from DESCRIPTION for now, as Roxygen and Travis didn't seem to mind and `demo(worked_steps)` still works.

File: man/plotunder.Rd [hunk 1]
```diff
@@ -24,13 +24,14 @@ N.B. using the mean with low levels of data can lead to spurious curves,
 the median appears to be more stable!
 }
 \examples{
-wafers_folder = paste(path.package("voltagefit"),"/extdata/",sep="") # path to wafters data directory
+wafers_folder = paste(path.package("voltagefit"),"/extdata/",sep="") # path to wafers data directory
```
Comment (csgillespie) [inline man/plotunder.Rd]: @extigy Would you change to `file.path` - platform independent.

File: man/undercurvesim.Rd [hunk 2]
```diff
@@ -20,7 +20,7 @@ A data.frame consisting of the fields:\cr
 Create sample used for underlying curves
 }
 \examples{
-wafers_folder = paste(path.package("voltagefit"),"/extdata/",sep="") # path to wafters data directory
+wafers_folder = paste(path.package("voltagefit"),"/extdata/",sep="") # path to wafers data directory
 fitted = fitall(wafers_folder)
 design = data.frame(week = c(1,1,1,1,2,2), wafer = unique(fitted$id), replicate = c(1:6), treatment = rep(1:6))
```
Comment (csgillespie) [inline man/undercurvesim.Rd]: `c(1:6) = rep(1:6) = 1:6` @extigy

File: R/data_help_files.R [hunk 3]
```diff
@@ -0,0 +1,47 @@
+## Entry in data_help_files.R
```
Comment (csgillespie) [inline R/data_help_files.R:1]: @extigy The help file entries can be condensed into a single entry. I think it's the `aliases` tag?

File: R/model.R [hunk 4]
```diff
@@ -1,22 +1,30 @@
...
+#'
+#' Fits logcurves to a wafer and returns the parameters and cost for both forward and backwards curves. 
+#' It is assumed that any wafer entering this function has been pre-filtered. 
+#' See the function "filtered" for details of how wafers are filtered.
+#'
+#' @param wafer     Data from a single wafer.
+#' @param iterlim   Maxiumum number of iterations to use in nlm (Default: 2000).
+#' @param id        WaferID (Default: A random string is generated as the ID).
+#' 
+#' @return A data.frame consisting of the fields:\cr
```
Comment (csgillespie) [inline R/model.R]: @extigy I don't think this will format correctly. You need to use a list.

File: R/model.R [hunk 5]
```diff
@@ -1,22 +1,30 @@
...
+#'
+#' @examples
+#' fitwafer(wafer3737)
+#'
+#' @importFrom stringi stri_rand_strings
+#' @importFrom stats lm manova nlm vcov
+#' @importFrom MASS mvrnorm
+#' 
+#' @export
 fitwafer = function(wafer, iterlim=2000, id=NULL){
```
Comment (csgillespie) [inline R/model.R]: @extigy Would you change all function names to have form: `fit_wafer`

**Pull request merged**